pybossa/sched.py

```diff
@@ -63,12 +63,34 @@ def new_task(project_id, sched, user_id=None, user_ip=None,
         Schedulers.user_pref: get_user_pref_task,
         'depth_first_all': get_depth_first_all_task}
     scheduler = sched_map.get(sched, sched_map['default'])
-    present_gold_task = not random.randint(0, 10)
 
-    return scheduler(project_id, user_id, user_ip, external_uid,
-                     offset=offset, limit=limit, orderby=orderby, desc=desc,
-                     rand_within_priority=rand_within_priority,
-                     present_gold_task=present_gold_task)
+    gold_only = False
```

Comment:
This logic seems more appropriate for [this function](https://github.com/bloomberg/pybossa/blob/master/pybossa/api/__init__.py#L182)